### DISCUSSION TITLE
【输出 GPT 总结的内容（仅自建）】可以只要总结后的内容。不要原始文章吗

### DISCUSSION THREAD

**Discussion by Whichbfj28** [DIYgod/RSSHub#16212] [Q&A]
### 这是一个什么样的功能？

仅保留总结后的提要。
https://docs.rsshub.app/zh/guide/parameters#%E8%BE%93%E5%87%BA-gpt-%E6%80%BB%E7%BB%93%E7%9A%84%E5%86%85%E5%AE%B9-%E4%BB%85%E8%87%AA%E5%BB%BA

### 这个功能可以解决什么问题？

减少阅读量

### 这不是重复的功能请求和 RSS 提案

- [X] 我已经搜索了 [现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保这项功能尚未被请求及不是 [新的 RSS 提案](https://github.com/DIYgod/RSSHub/issues/new?assignees=&labels=RSS+proposal&template=rss_request_zh.yml)。

**Comment by TonyRL** [comment]
You can update the prompt via environment variables. 
https://github.com/DIYgod/RSSHub/blob/eb4f019483d52d6f5f147fa31dc5a8b568d6eadd/lib/config.ts#L436

**Reply by Whichbfj28** [reply]
> You can update the prompt via environment variables.
> 
> https://github.com/DIYgod/RSSHub/blob/eb4f019483d52d6f5f147fa31dc5a8b568d6eadd/lib/config.ts#L436

我想问下。你是机器人吗？。
